attestation/github/github.go

```diff
@@ -89,34 +94,38 @@ func New() *Attestor {
 	}
 }
 
+// Name returns the name of the attestor.
 func (a *Attestor) Name() string {
 	return Name
 }
 
+// Type returns the type of the attestor.
 func (a *Attestor) Type() string {
 	return Type
 }
 
+// RunType returns the run type of the attestor.
 func (a *Attestor) RunType() attestation.RunType {
 	return RunType
 }
 
+// Attest performs the attestation for the github environment.
 func (a *Attestor) Attest(ctx *attestation.AttestationContext) error {
 	if os.Getenv("GITHUB_ACTIONS") != "true" {
 		return ErrNotGitlab{}
 	}
 
 	jwtString, err := fetchToken(a.tokenURL, os.Getenv("ACTIONS_ID_TOKEN_REQUEST_TOKEN"), "witness")
 	if err != nil {
-		return err
+		return fmt.Errorf("error on fething token %w", err)
 	}
 
 	spew.Dump(jwtString)
 
 	if jwtString != "" {
 		a.JWT = jwt.New(jwt.WithToken(jwtString), jwt.WithJWKSUrl(a.jwksURL))
 		if err := a.JWT.Attest(ctx); err != nil {
-			return err
+			return fmt.Errorf("error on attesting jwt %w", err)
```

Comment:
This is perhaps a larger conversation, but in other areas I've tried to use the wording "failed to ..." when wrapping errors. This isn't a deal breaker here but just a mental note that I think the maintainers should make a pass at language consistency in the library.

Reply:
I agree.